templates/lib/utils.js

```diff
@@ -0,0 +1,3 @@
+exports.getClassNameString = function() {
```

Comment:
this seems really poorly named. If the only thing this does is join any iterable with spaces, then let's not call it `getClassNameString`, or a file called `utils.js`, neither of which reflects what it's actually doing =)